Neos.TimeableNodeVisibility/Classes/Command/TimeableNodeVisibilityCommandController.php

```diff
@@ -28,21 +28,19 @@ public function executeCommand(string $contentRepository = 'default', bool $quie
             $this->output->outputLine(sprintf('Enabled %d nodes with exceeded timed dates.', $handlingResult->countByType(ChangedVisibilityType::NODE_WAS_ENABLED)));
             foreach ($handlingResult->getByType(ChangedVisibilityType::NODE_WAS_ENABLED) as $result) {
                 $this->output->outputLine(sprintf(
-                        '- NodeAggregateId: %s, DimensionSpacePoint: %s, Label: %s',
+                        '- NodeAggregateId: %s, DimensionSpacePoint: %s',
                         $result->node->nodeAggregateId->value,
-                        join(',', $result->node->originDimensionSpacePoint->coordinates),
-                        $result->node->getLabel()
```

Comment:
why can't we just keep rendering the label here (and below)?

Reply:
it was a new thing introduced by denny. We dont use the label for this in other places and it become a bigger burden now that one needs a service. Not saying we def should not do it but if we start using the label in exception we should do so consistently and not 3 times randomly in this new timeable node package.

Reply:
okay, fine by me ^^